### PR TITLE
30 playback

### PR DESCRIPTION
**Pull request by bameyrick**
Claiming Jon's work as my own :kratz:

### CODE REVIEW

**Comment by bameyrick** [comment]
[Playback](https://trello.com/c/KKKCB4Xu/30-playback)

File: src/patterns/playback/index.njk [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Check answers
+group: Help users to...
+---
+
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+Allow users to check the information they have have previously entered and confirm that it is correct before submitting it.
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Typo

File: src/patterns/playback/index.njk [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+A summary of multiple answers to a single question.
+
+{{
+    patternlibExample({ "path": "patterns/playback/examples/summary-multiple/index.njk" })
+}}
+
+A summary page should be displayed at the end of a section or at the end of a whole survey.
+
+<figure>  
+  <img src="../img/playback-summary.svg" alt="The playback of answer(s) within a summary">
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Broken image

File: src/patterns/playback/index.njk [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  <figcaption>Figure 1. An example where the respondent has completed a section or survey and their answers are played back within a summary.</figcaption>
+</figure>
+
+### 2. The playback of a single answer
+<figure>
+  <img src="../img/playback-singular-answer.svg" alt="The playback of a singular answer">
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Broken image

File: src/patterns/playback/index.njk [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  <figcaption>Figure 2. An example where the respondent is required to confirm a single answer in order to progress.</figcaption>
+</figure>
+
+### 3. The playback of multiple answers or a list of options
+<figure>
+  <img src="../img/playback-multiple-answers.svg" alt="The playback of multiple answers or a list of options">
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Brokem image

File: src/patterns/playback/index.njk [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  <figcaption>Figure 3. An example where the respondent is required to confirm multiple answers or a list of options previously entered.</figcaption>
+</figure>
+
+
+## Research on this component
+
+{% from "components/panel/src/_macro.njk" import onsPanel %}
+{{ 
+    onsPanel({
+        "body": "If you have conducted any user research using this pattern, please feed back your findings via the <a href='#'>Design System Forum.</a>"
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Path required for link

File: src/patterns/playback/index.njk [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    }) 
+}}
+
+## Design System forum
+
+[Discuss 'Check answers' on GitHub](#)
```
Comment (boxadesign) [inline src/patterns/playback/index.njk]: Path required for link

**Comment by jrbarnes9** [comment]
screenshot 2019-01-28 at 11 49 19
I know this is a comment on the summary component, but I think the line height of the question should be the same as the answer.  In the attached, the text area has a greater line height than the question. I think the question line height is probably too tight.  It's also tighter than in v1, so may not match the design any more.

**Pull request merged**